wpilib/wpilib/robotdrive.py

```diff
@@ -86,6 +88,7 @@ def __init__(self, *args, **kwargs):
         numbers are passed, the motorController keyword argument, if present,
         is the motor controller class to use; if unspecified, :class:`.Talon` is used.
         """
+        warnings.warn("Deprecated in 2018.0.0. Use DifferentialDrive, KilloughDrive, or MecanumDrive instead", DeprecationWarning)
```

Comment:
You should tell the user _what_ is deprecated, and where the deprecated code is being used (with the `stacklevel` argument).